packages/beeq/src/components/switch/scss/bq-switch.scss

```diff
@@ -44,7 +44,7 @@
 }
```

Comment:
For proper functionality when writing-mode: vertical-rl is used, 
this transition should be modified to apply `translate-y-[calc(var(--bq-switch--width)_-_var(--bq-switch--dot-size)_-_8px)] instead`.  Should we build a special case to support this, or leave it to the developers' discretion? @dgonzalezr @endv-bogdanb Any thoughts?

Reply:
We just removed translate and used `inset-inline-start` instead, this will work for LTR and RTL layouts properly.